tests/unit/CMakeLists.txt

```diff
@@ -20,6 +22,11 @@ target_include_directories(test_geos_unit
   PRIVATE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>)
 
+# Precompile TUT
+#if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.16")
+#    target_precompile_headers(test_geos_unit PRIVATE "${CMAKE_CURRENT_LIST_DIR}/tut/tut.hpp")
+#endif()
+
 file(GLOB_RECURSE _testfiles ${CMAKE_CURRENT_LIST_DIR}/**/*Test.cpp CONFIGURE_DEPEND)
```

Comment:
On a related note I'm considering a PR to increase the minimum CMake version to 3.16 (same as PROJ and GDAL). Xref #936